docs/guide/installation.rst

```diff
@@ -0,0 +1,54 @@
+************
+Installation
+************
+
+The SunPy project `maintains a range of packages <https://sunpy.org/project/affiliated>`__ that leverage the wider ecosystem of scientific Python packages for solar physics.
+
+conda
+=====
+
```

Comment:
Maybe (given the cross-link from the tutorial) here is a good place to explain why we recommend conda & miniforge in particular as the default option?

Reply:
👍 done, but suggestions for better text very welcome